app/components/RequestQuote.tsx

```diff
@@ -0,0 +1,97 @@
+// LIBRARY IMPORTS
+import { Box, VStack, Text, HStack, Icon } from "@chakra-ui/react";
+import { ReactNode } from "react";
+import { FaRegClipboard, FaRegEnvelope, FaRegHandshake } from "react-icons/fa6";
+import Image from "next/image";
+import { PageBlocksQuoteSection } from "@/tina/__generated__/types";
+import { TinaMarkdown } from "tinacms/dist/rich-text";
+
+// LOCAL IMPORTS
+import QuoteForm from "./QuoteForm";
+import machineBackground from "../../public/images/machineBackground.png";
+
+export default function RequestQuote(props: PageBlocksQuoteSection) {
+  return (
+    <Box position="relative">
+      <Image
+        src={machineBackground}
+        alt=""
+        placeholder="blur"
+        fill
+        sizes="100vw"
+        style={{
+          objectFit: "cover",
+          zIndex: -1,
+        }}
+      />
+      <HStack
+        px={10}
+        py={20}
+        spacing={16}
+        justifyContent="space-around"
+        mx="auto"
+        bgImage={machineBackground}
+        bg="rgba(11, 17, 62, 0.79)"
+      >
+        <Box
+          bg="white"
+          flex="1"
+          minW={{ base: "auto", md: "500px" }}
+          maxW={{ base: "auto", lg: "500px" }}
+        >
+          <QuoteForm {...props.requestQuoteForm} />
+        </Box>
+        <VStack flex="1" maxW={{ base: "auto", lg: "500px" }} spacing={4}>
+          <Text fontSize="3xl" fontWeight="bold" color="white" mb={6}>
+            {props.requestCopyTitle}
+          </Text>
+          <HStack align="flex-start">
+            <Icon as={FaRegClipboard} boxSize={8} color="brand.button" mr={2} />
+            <Box>
+              <TinaMarkdown
+                content={props.fillFormCopy}
+                components={{
+                  p: ({ children }: { children: ReactNode }) => (
+                    <Text as="span" color="white">
+                      {children}
+                    </Text>
+                  ),
+                }}
+              />
+            </Box>
+          </HStack>
+          <Text color="white" fontSize="3xl">
+            OR
+          </Text>
+          <HStack align="flex-start">
+            <Icon as={FaRegEnvelope} boxSize={8} color="brand.button" mr={2} />
+            <Box>
+              <TinaMarkdown
+                content={props.emailCopy}
+                components={{
+                  p: ({ children }: { children: ReactNode }) => (
+                    <Text as="span" color="white">
+                      {children}
+                    </Text>
+                  ),
+                }}
+              />
```

Comment:
For this `<TinaMarkdown />` component, I used essentially the same exact syntax as lines 57-66 of `CompanyValues.tsx`, which successfully renders on two lines and each line is formatted differently, which is reflected in the rendering. For some reason it's not working here and I can't figure out why 🤔 😕

Reply:
I think the issue here is with you rendering the whole markdown as a span. Since for each section you essentially have a heading and a paragraph, that's how you're supposed to be setting up your markdown for it to render accordingly. You can change it either in the home.md or in the visual editor as I mentioned in Slack

Reply:
Good catch! I followed a sample project from TinaCMS and found how you can render each section of the rich-text based on whether its h1, p, etc. The only limitation is that the client cannot toggle between sizes in the visual editor because then the text defaults to the default dark blue text color which can't be read on the blue background. I don't know if there's a way to apply the color white prop to the whole TinaMarkdown component? But I think it works well enough right now so I didn't spend too much time on that minor limitation

Reply:
I've added this as a task to our backlog so we can update all field names at once to make them more descriptive